app/models/profile.rb

```diff
@@ -24,6 +24,8 @@ class Profile < VersionedModel
 
   before_validation :set_assessment_answers
 
+  # TODO: uncomment this when category attribute is removed from model
+  # belongs_to :category, optional: true
```

Comment:
@martyn-w I had to comment this for now as it conflicts with the existing `category` attribute.

Reply:
👍  sure